src/v2/ion/uiSchemaLabelTransformer.js

```diff
@@ -0,0 +1,143 @@
+  'identify.identifier': 'primaryauth.username.placeholder',
+  'identify.credentials.passcode': 'primaryauth.password.placeholder',
+  'identify.rememberMe': 'remember',
+
+  'select-authenticator-enroll.authenticator.email': 'oie.email',
+  'select-authenticator-enroll.authenticator.password': 'oie.password',
+  'select-authenticator-enroll.authenticator.phone': 'oie.phone',
+  'select-authenticator-enroll.authenticator.security_key': 'oie.webauthn',
+  'select-authenticator-enroll.authenticator.security_question': 'oie.security.question',
+
+  'authenticator-enrollment-data.phone.authenticator.phoneNumber': 'mfa.phoneNumber.placeholder',
+
+  'enroll-authenticator.password.credentials.passcode': 'oie.password.passwordLabel',
+  'enroll-authenticator.security_question.sub_schema_local_credentials.0': 'oie.security.question.questionKey.label',
+  'enroll-authenticator.security_question.sub_schema_local_credentials.1': 'oie.security.question.createQuestion.label',
+  'enroll-authenticator.security_question.credentials.answer': 'mfa.challenge.answer.placeholder',
+  'enroll-authenticator.security_question.credentials.question': 'oie.security.question.createQuestion.label',
+  'enroll-authenticator.security_question.credentials.questionKey': 'oie.security.question.questionKey.label',
+
+  'select-authenticator-authenticate.authenticator.email': 'oie.email',
+  'select-authenticator-authenticate.authenticator.password': 'oie.password',
+  'select-authenticator-authenticate.authenticator.phone': 'oie.phone',
+  'select-authenticator-authenticate.authenticator.security_key': 'oie.webauthn',
+  'select-authenticator-authenticate.authenticator.security_question': 'oie.security.question',
+
+  'challenge-authenticator.email.credentials.passcode': 'mfa.challenge.enterCode.placeholder',
+  'challenge-authenticator.password.credentials.passcode': 'mfa.challenge.password.placeholder',
+  'challenge-authenticator.phone.credentials.passcode': 'mfa.challenge.enterCode.placeholder',
+  'challenge-authenticator.security_question.credentials.answer': 'mfa.challenge.answer.placeholder',
+};
+
+const getI18nKey = (i18nPath) => {
+  let i18nKey;
+  if (i18nPath.indexOf(SECURITY_QUESTION_PREFIX) === 0) {
+    const securityQuestionValue = i18nPath.replace(SECURITY_QUESTION_PREFIX, '');
+    i18nKey = `security.${securityQuestionValue}`;
+  }
+
+  if (I18N_OVERRIDE_MAPPINGS[i18nPath]) {
+    i18nKey = I18N_OVERRIDE_MAPPINGS[i18nPath];
+  }
+
+  if (i18nKey && !Bundles.login[i18nKey]) {
+    Logger.warn(`expect i18n key ${i18nKey} for ${i18nPath} but not found in 'login' bundle.`);
+  }
+
+  return i18nKey;
+};
+
+const overrideLabelIfI18NPathExists = (i18nPath, value) => {
+  const i18nKey = getI18nKey(i18nPath);
+  if (i18nKey) {
+    return loc(i18nKey, 'login');
+  } else {
+    return value;
+  }
+};
+
+const updateLabelForUiSchema = (remediation, uiSchema) => {
+  if (uiSchema.mutable === false) {
+    return;
+  }
+  Logger.info('i18n label transformer');
```

Comment:
Do we need to expose all this logger messages in console ?

Reply:
It only shows up on `dev` bundle (`yarn build:dev`) so probably fine ?